README.md

```diff
@@ -40,6 +40,10 @@ docker run -it --rm ghcr.io/mia-platform/mlp
 go get -u github.com/mia-platform/mlp
 ```
 
+## Test
+
+Run tests using `make test` in the repo root directory. Notice that in order to run integration tests locally you **must have** configured a k8s cluster connection in the running environment.
```

Comment:
This is not very comfortable. If this is a requirement, we should create and setup a kind cluster all in the test command.
Furthermore, to run tests only on cluster also locally, the risk is to run the tests in a remote environment where we are already connected

Reply:
Same as [this](https://github.com/mia-platform/mlp/pull/29#discussion_r878252076)